omod/src/main/java/org/openmrs/module/fhir/providers/RestfulGroupResourceProvider.java

```diff
@@ -0,0 +1,46 @@
+package org.openmrs.module.fhir.providers;
+
+import ca.uhn.fhir.rest.annotation.IdParam;
+import ca.uhn.fhir.rest.annotation.Read;
+import ca.uhn.fhir.rest.annotation.RequiredParam;
+import ca.uhn.fhir.rest.annotation.Search;
+import ca.uhn.fhir.rest.param.StringParam;
+import ca.uhn.fhir.rest.param.TokenParam;
+import ca.uhn.fhir.rest.server.IResourceProvider;
+import org.hl7.fhir.dstu3.model.Group;
+import org.hl7.fhir.dstu3.model.IdType;
+import org.hl7.fhir.instance.model.api.IBaseResource;
+import org.openmrs.module.fhir.resources.FHIRGroupResource;
+
+import java.util.List;
+
+public class RestfulGroupResourceProvider implements IResourceProvider {
```

Comment:
why there are no create or update methods here?

Reply:
At beginning of work, I didn't realized that they are required because there wasn't any in class that I was relying on. Now I am not able to map Group to Cohort. Field "memberships" can't be set. 
If I try to set CohortMembership memberships then I get "ClassNotFoundException: org.openmrs.CohortMembership" exception. 
But if I will try to use addMember(Integer memberId) then I get "NoSuchMethodError: org.openmrs.Cohort.addMember(Ljava/lang/Integer;)" exception.

Reply:
Okay, never mind my comment. I found a solution. I will fix it soon.